test/models/fake_auth_test.rb

```diff
@@ -15,16 +15,7 @@ class FakeAuthTest < ActiveSupport::TestCase
   test 'fakeauth enabled pr pattern app name' do
     ClimateControl.modify(
       FAKE_AUTH_ENABLED: 'true',
-      HEROKU_APP_NAME: 'thesis-dropbox-pr-123'
-    ) do
-      assert_equal(true, fake_auth_status)
-    end
-  end
-
-  test 'fakeauth enabled pr minus s pattern app name' do
```

Comment:
Please check me on this - I believe that now, with only the `thesis-submit-pr-#` naming convention, we only need one test here. From the name, and the history of this test, it seems like we can delete this because `...pr-s-` is no longer a pattern we might encounter.

Reply:
yeah we can control PR name patterns more now than we could then so I do suspect this is no longer relevant.

Reply:
I've not restored this test yet, as I think it is still not needed even with the old and new pipelines being tested. If you'd like this back too, I can add it.

Reply:
No you are correct, this still remains unneeded.